tycho-bundles/org.eclipse.tycho.core.shared/src/main/java/org/eclipse/tycho/core/shared/MultiLineLogger.java

```diff
@@ -88,6 +88,19 @@ public void debug(String message) {
         }
     }
 
+    @Override
+    public void debug(String message, Throwable cause) {
+        String[] messageLines = message.split("\n");
```

Comment:
what about `"\r?\n"` to catch both linux and windows?

Reply:
I just copied that from the existing implementations. If we want to change her we should change this for the other methods as well.
Beside that, the purpose of multi-line logger is a bit strange to me especially if it comes to parallel builds it could soon become quite confusing.